podman/api/http_utils.py

```diff
@@ -69,20 +67,21 @@ def prepare_body(body: MutableMapping[str, Any]) -> str:
 
 
 def _filter_values(mapping: Mapping[str, Any]) -> Dict[str, Any]:
-    """Returns a canonical dictionary with values == None or empty Iterables removed."""
+    """Returns a canonical dictionary with values == None or empty Iterables removed.
```

Comment:
```suggestion
    """Returns a canonical dictionary with values == None or empty. Iterables removed.
```

Reply:
Not this time, I am looking for empty Iterables. :-)